webapp/tests/karma/ts/modules/contacts/contacts-edit.component.spec.ts

```diff
@@ -287,7 +289,7 @@ describe('ContactsEdit component', () => {
         await createComponent();
         await fixture.whenStable();
 
-        expect(contactTypesService.get.callCount).to.equal(1);
+        expect(contactTypesService.get.callCount).to.equal(2);
```

Comment:
Ideally, we should only be getting the type once.

Reply:
Fixed by deduplicating the calls mentioned in the comment above